Packs/Expanse/Integrations/Expanse/Expanse.py

```diff
@@ -571,6 +762,82 @@ def domain_command():
     return_outputs(human_readable, ec, domain)
 
 
+def certificate_command():
+    """
+    searches by domain name for certificate information
+    """
+    search = demisto.args()['common_name']
+    params = {
+        "commonNameSearch": search
+    }
+    token = do_auth()
+    results = http_request('GET', 'assets/certificates', params, token=token)
+    try:
+        certs = results['data']
+    except Exception:
+        demisto.results("No data found")
+        return
+
+    cert = certs[0]  # just return the first one
+    cert['search'] = search
+
+    dbot_context = {
+        "Indicator": search,
+        "Type": "certificate",
+        "Vendor": "Expanse",
+        "Score": 0
+    }
+    cert_context = get_certificate_context(cert)
+    expanse_cert_context = get_expanse_certificate_context(cert)
+
+    ec = {
+        'DBotScore': dbot_context,
+        'Certificate(val.SearchTerm == obj.SearchTerm)': cert_context,
+        'Expanse.Certificate(val.SearchTerm == obj.SearchTerm)': expanse_cert_context
+    }
+    human_readable = tableToMarkdown("Certificate information for: {search}".format(search=search), expanse_cert_context)
+
+    return_outputs(human_readable, ec, cert)
+
+
+def behavior_command():
+    """
+    searches by ip for behavior details from Expanse
+    """
+    search = demisto.args()['ip']
+
+    now = datetime.today()
+    time_range = datetime.strftime(now - timedelta(days=FIRST_RUN), "%Y-%m-%d")
+    params = {
+        "filter[internal-ip-range]": search,
+        'page[limit]': 20,
+        'filter[created-after]': time_range + 'T00:00:00.000Z',
+    }
+    token = do_auth()
+    results = http_request('GET', 'behavior/risky-flows', params, token=token)
+    try:
+        behaviors = results['data']
+    except Exception:
+        demisto.results("No data found")
+        return
+
+    dbot_context = {
```

Comment:
no need for dbot_Score, we don't have reputation for behavior nor certificate. List of standard outputs https://xsoar.pan.dev/docs/integrations/context-standards